🧹: Removes forced newline WRAPs (#11)
psbanka/build-your-own-internet: chapters/001-getting-started/README.md

              (squasheeba 10.1.1.0/24)
```

> ## Tools you will need installed

Reviewer: We have [this section](https://github.com/psbanka/build-your-own-internet#pre-requisites) in the main README. Does that serve this purpose? I'm fine with it being duplicated/expanded here, but just wanted to check.

We will be using following tools in this exercise. These need to be installed on your machine (not within the docker container). You can run `which docker` and `which jq` first to see if they return a path to a binary. If they do, that means they are already installed. If they don't you can follow instructions below to install them.

> - docker: This is just the docker command like utility, you don't need more than that. If you use homebrew as your package manager on macOS machine, you can run `brew install docker`.

Reviewer: ```suggestion
- docker: This is just the docker command line utility, you don't need more than that. If you use homebrew as your package manager on a macOS machine, you can run `brew install docker`.
```

- jq: You can follow instructions on [this page](https://jqlang.github.io/jq/download/) to install jq if you don't have it on your local machine.

## Vocabulary

> - `container`: a running instance of a docker image. Each container we create in
>   building out our internet will be one machine on that internet.

Reviewer: ```suggestion
- `container`: a running instance of a docker image. Each container we create will be one machine on our internet.
```

64 bytes from 10.1.1.3: icmp_seq=3 ttl=64 time=0.240 ms
```

Basically, all you need to know about this is that `ping` is a program that sends packets across the network using a protocol called ICMP, which stands for Internet Control Message Protocol. `echo request` and `echo reply` are two types of ICMP message. You can read more about them [here](https://docs.netapp.com/us-en/e-series-santricity/sm-hardware/what-are-icmp-ping-responses.html) if you want to know more. What we see here in this ping message is that it has both sent a packet to the destination (echo request), and the destination has replied (echo reply). The `icmp_seq=3` designation marks each individual request/response pair. If the ping did not go through, you might see various error messages, but the most common is that the `ping` command replies with `Request timeout for icmp_seq 0` type messages.

> On the container that is being pinged (pippin), we see quite a bit more information from tcpdump. First, we see a series of messages that have `ARP` in them. ARP, which stands for Address Resolution Protoco,  is a protocol that allows a machine that is connected locally on one network to talk to another machine that is also connected to that same network (as opposed to a machine that wants to communicate over multiple networks). You can learn all about ARP [here](https://www.fortinet.com/resources/cyberglossary/what-is-arp) if you are interested in diving deeper.

Reviewer: Oh blergh. We cover this in part in the [IP and MAC address appendix](https://github.com/psbanka/build-your-own-internet/blob/004-ip-and-ethernet/chapters/appendix/ip-and-mac-addresses.md)... Maybe in addition to sending people out to the internet we could say that we'll discuss this a bit more in future chapters? So, like 

> We'll talk about this crucial protocol more over the coming chapters, but you can get a head start learning about ARP [here](https://www.fortinet.com/resources/cyberglossary/what-is-arp).

> After seeing the ARP packets go back and forth (which establish the ability for those two containers to talk to each other on the local network), we see the ICMP echo-request and echo-reply packets go back and forth in our `tcpdumnp` output.

Reviewer: ```suggestion
After seeing the ARP packets go back and forth (which establish the ability for those two containers to talk to each other on the local network), we see the ICMP echo-request and echo-reply packets go back and forth in our `tcpdump` output.
```
